tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -169,7 +169,7 @@ public DataFrame SplitTrainTest(DataFrame input, float testRatio, out DataFrame
         {
             IEnumerable<int> randomIndices = Enumerable.Range(0, (int)input.RowCount);
             IEnumerable<int> trainIndices = randomIndices.Take((int)(input.RowCount * testRatio));
-            IEnumerable<int> testIndices = randomIndices.TakeLast((int)(input.RowCount * (1 - testRatio)));
```

Comment:
Is this a netcore only API?

Reply:
Correct - it isn't available on .NET Framework.

https://apisof.net/catalog/System.Linq.Enumerable.TakeLast%3CTSource%3E(IEnumerable%3CTSource%3E,Int32)